server/sonar-server/src/test/java/org/sonar/server/platform/ws/MigrateDbActionTest.java

```diff
@@ -80,10 +84,13 @@ public void wireMocksTogether() {
     when(database.getDialect()).thenReturn(dialect);
   }
 
-  @Test(expected = IllegalStateException.class)
+  @Test
   public void ISE_is_thrown_when_version_can_not_be_retrieved_from_database() throws Exception {
     when(databaseVersion.getVersion()).thenReturn(null);
```

Comment:
same here, if my understanding is correct you don't need to specify that it returns null